omnibus/config/projects/chefdk.rb

```diff
@@ -89,6 +86,7 @@
 dependency "chef-provisioning-fog"
 dependency "chef-provisioning-vagrant"
 dependency "chef-provisioning-azure"
+dependency "ruby-windows-devkit" if windows?
```

Comment:
Yes...  dependency says "please pull this in now".  override says "if you pull this in, please make the properties blah" (regardless of if either project or property exists).  That's why we can override the :enabled property of the :fips software when neither of those exists